service/automatic_run_experiment.py

```diff
@@ -91,6 +91,12 @@ def _get_description(experiment_config: dict) -> Optional[str]:
     return experiment_config.get('description')
 
 
+def _use_oss_fuzz_corpus(experiment_config: dict) -> Optional[str]:
```

Comment:
Those arbitrary kwargs would need to be validated still, as otherwise run_experiment will fail ?
Lets maybe do this in another cl.

Reply:
I sorta think we might do too much validation in this project. I'm fine with not validating things here if they get validated in start_experiment.
But it can absolutely wait. Doing it now probably violates YAGNI

Reply:
You don't need to do it now, but I guess it would make sense to do this now since we now have oss_fuzz_corpora and description.

Reply:
Yes added validation now.

Reply:
Oh sorry, my comment was very ambigous. I was talking about the kwarg stuff.
I think the validation is nice but I wasn't suggesting doing it now. I'm not sure if I would have added validation myself since I think we may have too much validation code already leading to needless complexity. But it can stay for now. We can discuss validation later.